integrations/aws/main.py

```diff
@@ -0,0 +1,232 @@
+        async for batch in resync_cloudcontrol(kind):
+            yield batch
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.ACM)
+async def resync_acm(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in get_sessions():
+        async for batch in batch_resources(
+            kind,
+            session,
+            "acm",
+            "list_certificates",
+            "CertificateSummaryList",
+            "NextToken",
+        ):
+            yield batch
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.LOADBALANCER)
+async def resync_loadbalancer(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in get_sessions():
+        async for batch in batch_resources(
+            kind,
+            session,
+            "elbv2",
+            "describe_load_balancers",
+            "LoadBalancers",
+            "NextMarker",
+        ):
+            yield batch
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.CLOUDFORMATION)
+async def resync_cloudformation(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in get_sessions():
+        async for batch in batch_resources(
+            kind,
+            session,
+            "cloudformation",
+            "list_stacks",
+            "StackSummaries",
+            "NextToken",
+        ):
+            yield batch
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.EC2)
+async def resync_ec2(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in get_sessions():
+        region = session.region_name
+        account_id = await _session_manager.find_account_id_by_session(session)
+        try:
+            async with (
+                session.resource("ec2") as ec2,
+                session.client("ec2") as ec2_client,
+            ):
+                async for page in ec2.instances.pages():
+                    if not page:
+                        continue
+                    page_instances = []
+                    instance_ids = [instance.id for instance in page]
+                    described_instances = await ec2_client.describe_instances(
+                        InstanceIds=instance_ids,
+                    )
+                    for reservation in described_instances["Reservations"]:
+                        for instance in reservation["Instances"]:
+                            instance.update(
+                                {
+                                    KIND_PROPERTY: kind,
+                                    ACCOUNT_ID_PROPERTY: account_id,
+                                    REGION_PROPERTY: region,
+                                }
+                            )
+                            page_instances.append(
+                                fix_unserializable_date_properties(instance)
+                            )
+                    yield page_instances
+        except Exception:
+            logger.exception(f"Failed to list EC2 Instance in region: {region}")
```

Comment:
why are you not raising the error?
it will leed to missing entities that will be deleted after the resync